openpocket/alpha_spheres/alpha_spheres.py

```diff
@@ -126,27 +125,44 @@ def remove_alpha_spheres(self, indices):
         --------
 
         """
-
-        pass
-
-    def remove_small_alpha_spheres(self, minimum_radius):
-
-        """
-
-        """
-
-        indices_to_remove = np.where(self.radii < minimum_radius)
-        self.remove(indices_to_remove)
+        if indices is not None:
+        #    if isinstance(indices, (list,np.ndarray,dtype:ints)): 
+            if isinstance(indices, (list,np.ndarray)):
+                pass
+            else:
+                raise ValueError("The argument points needs to be a List, tuple or numpy.ndarray object with the integer numbers corresponding to the alpha-sphere")
+
+            self.indices = indices  # a way to store the input indices
```

Comment:
There is no need to define a new attribute named `indices` in the class. What would be its use? Why do you want to store the indices of the removed alpha-spheres? They will make no sense after removing them since the indices will be redefined. Example: you have indices [0,1,2,3] and you remove [0,1], the new set will have indices [0,1] (the former [2,3]).

Reply:
I agree, it was a mistake: there is no need to do it.